front/app/containers/Admin/projects/project/permissions/components/UserSearch.tsx

```diff
@@ -90,6 +107,7 @@ const UserSearch = memo(
             disabled: isModerator(user)
               ? get(user.attributes, 'is_moderator')
               : get(user.attributes, 'is_member'),
```

Comment:
```suggestion
              ? user.attributes['is_moderator']
              : user.attributes['is_member'],
```

Since we touch this file, let's get rid of lodash's `get`. It doesn't offer type warnings. (Try modifying get(user.attributes, 'is_moderator') to get(user.attributes, 'is_citizen') for example. It won't complain.

I'll also mention this in the FE CDM.